dbms/src/Flash/Coprocessor/DAGContext.h

```diff
@@ -378,6 +380,9 @@ class DAGContext
 
 private:
     std::shared_ptr<ProcessListEntry> process_list_entry;
+    /// Holding the table lock to make sure that the table wouldn't be dropped during the lifetime of this query,
```

Comment:
Comment out why the `table_locks` is inited before streams?
Then we can make the lifetime of each component more clear.

Reply:
ok

Reply:
done